Documentation for Quectel KG200ZABTB (#1345)
TheThingsIndustries/lorawan-stack-docs: doc/content/devices/models/quectel-kg200z/_index.md

---
title: "Quectel KG200ZABTB"
description: ""
weight: 
---

{{< figure src="kg200z.png" alt="Quectel KG200ZABTB" class="float plain" width="60%">}}

> The [Quectel KG200ZABTB](https://www.quectel.com/product/lora-kg200z) is a high-performance LoRa module launched by Quectel, which supports ultra-low power consumption and long-range wireless transmission applications. It integrates an ARM Cortex-M4 core with the modulations of LoRa,(G)FSK, (G)MSK and BPSK, supporting LoRaWAN standard protocol and 470–510MHz, 862–928MHz LoRa frequency bands. Additionally, it incorporates AES hardware encryption for enhanced security. KG200Z features a compact form factor of 12.0mm × 12.0mm × 1.8mm and an LGA package to ensure seamless embedding of the module into size-constrained applications and reliable connectivity for these applications.

Reviewer: ```suggestion
The [Quectel KG200ZABTB](https://www.quectel.com/product/lora-kg200z) is a high-performance LoRa module launched by Quectel, which provides ultra-low power consumption and long-range wireless transmission applications. It integrates an ARM Cortex-M4 core with the modulations of LoRa, (G)FSK, (G)MSK and BPSK, supporting LoRaWAN standard protocol, as well as 470–510MHz and 862–928MHz LoRa frequency bands. Additionally, it incorporates AES hardware encryption for enhanced security. KG200Z features a compact form factor of 12.0mm × 12.0mm × 1.8mm and an LGA package to ensure seamless embedding of the module into size-constrained applications and reliable connectivity for these applications.
```

> This evaluation board (EVK) allows for programming of the module itself. For this guide though, we are just focussing on connecting it to {{% tts %}}

Reviewer: What do you mean by "this evaluation board"? Which board? I only see described KG200Z module. Is there a link to the product? It would be also good to rephrase this a bit

Author: There is a Mouser page for it here: https://mouser.com/ProductDetail/Quectel/KG200ZABTB?qs=dbcCsuKDzFU6qVuvnq%252Bzmg%3D%3D This is the EVK

> This evaluation board (EVK) allows for programming of the module itself. For this guide though, we are just focussing on connecting it to {{% tts %}}

Reviewer: ```suggestion
In this guide, we focus on connecting the evaluation board (EVK) to {{% tts %}}. This board can be used for programming the KG200Z module.
```

> <!--more-->

Reviewer: This tag needs to be placed in a such manner that the card isn't too big and contains only section description. Right now it's too big 
![image](https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/65215579/78149253-d86f-4b47-aaf5-cd8a730b3145)

## Provisioning Information

> To get the provisioning information of the EVK we need to connect to the board via serial. The steps for connecting to it via serial depends on what OS you are using, so we have instructions for both [MacOS / Linux](#macos--linux) and [Windows](#windows).

Reviewer: ```suggestion
To get the provisioning information of the EVK we need to connect to the board via serial. The steps for connecting to it via serial depends on what OS you are using, [MacOS/Linux](#macos--linux) or [Windows](#windows).
```

### MacOS / Linux

> On MacOS and Linux, you can use the handy `screen` command that is built-in. First though, we need to know the address of our board. Before we continue though, **make sure your board is not yet plugged in**.

Reviewer: ```suggestion
On MacOS and Linux, you can use the handy built-in `screen` command.

First step is to find out the board address.
```

> Open up a terminal, and type `ls /dev/tty.*`. You should see a couple of addresses show up.

Reviewer: ```suggestion
Before plugging in the board, open up a terminal and type `ls /dev/tty.*`. You should see a couple of addresses show up.
```

{{< figure src="cmd-ls.png" class="plain    ">}}

Then plug in the EVK and turn it on. When you try the command again you should see a new address show up.

{{< figure src="cmd-ls-dev.png" class="plain">}}

> Now we are going to use the `screen` command with that address to connect to it. The command you are going to use is `screen <DEVICE-ADDRESS> 115200`.

Reviewer: ```suggestion
Now use the newly discovered board address to connect to the board using the `screen <DEVICE-ADDRESS> 115200` command.
```

{{< figure src="cmd-screen.png" class="plain">}}

> Once you do that, you should see some text show up. Now press the reset button on the EVK to have the device show it's provisioning information. Select and copy the **AppKey, DevEUI and AppEUI** and store them somewhere.

Reviewer: ```suggestion
Once you do that, you should see some text show up in your terminal. Now press the reset button on the EVK to have the device show its provisioning information. Select and copy the **AppKey**, **DevEUI** and **AppEUI** and store them somewhere.
```

{{< figure src="prov-info.png" class="plain">}}

> Using these credentials we can now [onboard it to {{% tts %}}](#onboarding-to--tts-).

Reviewer: ```suggestion
Using these parameters we can now proceed with [onboarding the device to {{% tts %}}](#onboarding-to--tts-).
```

### Windows 

> To connect to the EVK using Windows we need to install a third party tool that can interface with the EVK. Before we do that though, **make sure your board is not yet plugged in**.

Reviewer: ```suggestion
To connect to the EVK using Windows, a third party tool that can interface with the EVK needs to be installed first. Make sure your board isn't yet connected to your computer.
```

> For this tutorial we are going to use [SimpleCom](https://github.com/YaSuenag/SimpleCom/releases/latest), but any UART serial tool should do the trick. Download the ZIP file, unzip it and run **SimpleCom.exe**.

Reviewer: ```suggestion
For this tutorial we use [SimpleCom](https://github.com/YaSuenag/SimpleCom/releases/latest), but any UART serial tool should do the trick. Download the ZIP file, unzip it and run **SimpleCom.exe**.
```

> When you open it you will see this screen. If you select the COM dropdown, you will probably see a couple of COM ports. 

Reviewer: ```suggestion
When you run the tool, you will see the following screen. If you select the COM dropdown, you will probably see a couple of COM ports. 
```

{{< figure src="win-com.png" class="plain    ">}}

> Now close the program (using the **Cancel** button), then plug in the EVK and turn it on. Run the program again, and when you select the dropdown again you should see a new COM port show up. Select that one and make sure the **Baud Rate is set at 115200**.

Reviewer: ```suggestion
Now close the program (using the **Cancel** button), then plug in the EVK and turn it on. Run the program again, and when you select the dropdown again you should see a new COM port show up. Select that one and make sure the **Baud Rate** is set at `115200.
```

{{< figure src="win-com-dev.png" class="plain">}}

> Once that is done, hit **Connect** and you should see some text show up in the terminal. Now press the reset button on the EVK to have the device show it's provisioning information. Select and copy the **AppKey, DevEUI and AppEUI** and store them somewhere.

Reviewer: ```suggestion
Once that is done, hit **Connect** and you should see some text show up in the terminal. Now press the reset button on the EVK to have the device show its provisioning information. Select and copy the **AppKey**, **DevEUI and **AppEUI** and store them somewhere.
```

{{< figure src="win-com-prov.png" class="plain">}}

> Using these credentials we can now [onboard it to {{% tts %}}](#onboarding-to--tts-).

Reviewer: ```suggestion
Using these parameters we can now proceed with [onboarding the device to {{% tts %}}](#onboarding-to--tts-).
```

{{< note "Quectel KG200ZABTB cannot be onboarded to {{% tts %}} using QR code." />}}

## Onboarding to {{% tts %}}

> The Quectel KG200ZABTB has to be onboarded **manually**.

Reviewer: Link https://www.thethingsindustries.com/docs/devices/adding-devices/manual/
